app/domain/env.ts

```diff
@@ -20,10 +20,8 @@ export const PUBLIC_URL = (
   ""
 ).replace(/\/$/, "");
 
-export const SPARQL_ENDPOINT =
-  clientEnv?.SPARQL_ENDPOINT ??
-  process.env.SPARQL_ENDPOINT ??
-  "https://int.lindas.admin.ch/query";
+export const SPARQL_ENDPOINT = "https://int.lindas.admin.ch/query";
```

Comment:
Please revert this change :) If you want to configure this differently, set it in `.env.development` or `.env.local`

Reply:
This is only for the local dev purposes, I have a point in TODO to revert this 😆 I just needed to change the endpoint there to access proper datasets, but will revert after the feature is ready.

Reply:
I know but seems like it's easy to forget :) If you want to change it for testing, I recommend using the `.env` files https://nextjs.org/docs/basic-features/environment-variables